src/frontend/src/handler/describe.rs

```diff
@@ -179,6 +186,11 @@ pub fn handle_describe(handler_args: HandlerArgs, table_name: ObjectName) -> Res
                     DataType::Varchar.to_oid(),
                     DataType::Varchar.type_len(),
                 ),
+                PgFieldDescriptor::new(
```

Comment:
Need to add it as well in `infer` function in `src/frontend/src/session.rs`, which is used by extended mode.

Reply:
FYI, the failure of e2e test must be caused by it.